lib/occupier/postgres/client.rb

```diff
@@ -60,6 +62,8 @@ def drop_all
       # Connects to the database
       # when no database name is provided uses the database from the configuration
       def connect_to(database_name)
+        ActiveRecord::Base.establish_connection(db_config) unless ActiveRecord::Base.connected?
```

Comment:
There is no connection when running this in tests locally

Reply:
Wasn't the performance being affected?